rskj-core/src/main/java/co/rsk/peg/constants/BridgeConstants.java

```diff
@@ -106,6 +107,8 @@ public Coin getMinimumPeginTxValue(ActivationConfig.ForBlock activations) {
 
     public Coin getSpendableValueFromProposedFederation() { return minimumPegoutTxValue.multiply(2); }
 
+    public Keccak256 getProposedFederationFlyoverPrefix() { return new Keccak256("0000000000000000000000000000000000000000000000000000000000000001"); }
```

Comment:
Could you tell me what the reason is for this? Why do we need this prefix to send the funds to the proposed federation? Isn't enough the address of the P2shScript?